plugins/internal/nexus-atlas-plugin/src/main/java/org/sonatype/nexus/atlas/internal/SystemInformationGeneratorImpl.groovy

```diff
@@ -109,14 +109,25 @@ class SystemInformationGeneratorImpl
     def reportFileStores = {
       def data = [:]
       def fs = FileSystems.default
+      def ignoreIOex = { def defaultValue, Closure callMe ->
+        try {
+          return callMe.call()
+        }
+        catch (e) {
+          if (!e.class.isAssignableFrom(IOException.class)) {
+            throw e
+          }
+          return defaultValue
+        }
+      }
```

Comment:
simplify this to just eat all exceptions, no reason to special case IOException here.
probably log trace the exception as well.